apps/rush-lib/src/api/Rush.ts

```diff
@@ -102,16 +103,17 @@ export class Rush {
   private static _printStartupBanner(isManaged: boolean): void {
     const nodeVersion: string = process.versions.node;
     const nodeReleaseLabel: string = (NodeJsCompatibility.isOddNumberedVersion)
-    ? 'unstable'
-    : (NodeJsCompatibility.isLtsVersion ? 'LTS' : 'pre-LTS');
+      ? 'unstable'
+      : (NodeJsCompatibility.isLtsVersion ? 'LTS' : 'pre-LTS');
 
     console.log(
       EOL +
       colors.bold(`Rush Multi-Project Build Tool ${Rush.version}` + colors.yellow(isManaged ? '' : ' (unmanaged)')) +
-      colors.cyan(` - ${RushConstants.rushWebSiteUrl}`) +
+      colors.cyan(` - ${RushConstants.rushWebSiteUrl}`) + `.` +
       EOL +
-      `Node.js version is ${nodeVersion} (${nodeReleaseLabel})` +
-      EOL
+      `Node.js version is ${nodeVersion} (${nodeReleaseLabel}).` +
+      EOL + EOL +
+      `Rush is running in ` + colors.yellow(Utilities.isCI() ? `CI` : `NON-CI`) + ` mode.`
```

Comment:
I think there should be a few logging scenarios:

| `is-ci` detects CI? | `RUSH_CI_MODE` set to? | logging output |
| ---- | ---- | ---- |
| no | \<unset\> | \<nothing\> |
| yes | \<unset\> | `Rush is running in CI mode. To override, set the "RUSH_CI_MODE" environment variable to "false"` |
| yes | `true` | `Rush is running in CI mode. To override, set the "RUSH_CI_MODE" environment variable to "false"` |
| yes | `false` | `Rush detected a CI environment, but is running in NON-CI mode because the ""RUSH_CI_MODE" environment variable is set to "false"` |
| no | `false` | \<nothing\> |
| no | `true` | `Rush is running in CI mode because the "RUSH_CI_MODE" environment variable is set to "true"` |

Reply:
I will refactor the CI mode determination into a class and add a method "printWhy" or something that can print this output. I don't want other parts of Rush to be able to differentiate between OverrideCI and DetectedCI. Thoughts?